cmd/cluster/dr/switchover_dr.go

```diff
@@ -0,0 +1,91 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package dr
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	"github.com/yugabyte/ybm-cli/internal/formatter"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+var switchoverDrCmd = &cobra.Command{
+	Use:   "switchover",
+	Short: "Switchover DR for a cluster",
+	Long:  `Switchover DR for a cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+		authApi.GetInfo("", "")
+
+		drName, _ := cmd.Flags().GetString("dr-name")
+		clusterId, err := authApi.GetClusterIdByName(ClusterName)
```

Comment:
What's `ClusterName` here? Where does it come from?

Reply:
NVM I see it's in the args. Do we need it there though? Isn't DR config name enough?

Reply:
It comes from `--cluster-name` that we provide in the command line.

Reply:
Yes I see but isn't cluster name redundant still?

Reply:
I have designed the DR config to be a sub resource of the cluster resource, hence the cluster name needs to be given similar to the API.